sqlserver/README.md

```diff
@@ -69,7 +69,16 @@ To configure this check for an Agent running on a host:
 
     See the [example check configuration][6] for a comprehensive description of all options, including how to use custom queries to create your own metrics.
 
-    **Note**: The (default) provider `SQLOLEDB` is being deprecated. To use the newer `MSOLEDBSQL` provider, set the `adoprovider` variable to `MSOLEDBSQL` in your `sqlserver.d/conf.yaml` file after having downloaded the new provider from [Microsoft][7]. It is also possible to use the Windows Authentication and not specify the username/password with:
+For hosts that are configured with the `<hostname\instance name>`, add `reported_hostname:` to the `sqlserver.d/conf.yaml` file, in the `conf.d/` at the root of the [Agent's configuration directory][5]:
+
+```yaml
+init_config:
+instances:
+ - host:<host>\<name instance>
+   reported_hostname:<host>\<name instance> # add this line
+```
+
+   **Note**: The (default) provider `SQLOLEDB` is being deprecated. To use the newer `MSOLEDBSQL` provider, set the `adoprovider` variable to `MSOLEDBSQL` in your `sqlserver.d/conf.yaml` file after having downloaded the new provider from [Microsoft][7]. It is also possible to use the Windows Authentication and not specify the username/password with:
```

Comment:
What's the timeline on the deprecation? Are we the ones deprecating it? We should try to avoid vague temporal language here, I would suggest removing the word "being"
```suggestion
   **Note**: The (default) provider `SQLOLEDB` is deprecated. To use the newer `MSOLEDBSQL` provider, set the `adoprovider` variable to `MSOLEDBSQL` in your `sqlserver.d/conf.yaml` file after having downloaded the new provider from [Microsoft][7]. It is also possible to use the Windows Authentication and not specify the username/password with:
```

Reply:
Note, in 7.42 the default ADO provider will be MSOLEDBSQL  anyways https://github.com/DataDog/integrations-core/blob/master/sqlserver/datadog_checks/sqlserver/connection.py#L141